CHANGELOG.md

```diff
@@ -55,6 +55,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (x/auth) [#13210](https://github.com/cosmos/cosmos-sdk/pull/13210) Add `Query/AccountInfo` endpoint for simplified access to basic account info.
 * (x/consensus) [#12905](https://github.com/cosmos/cosmos-sdk/pull/12905) Create a new `x/consensus` module that is now responsible for maintaining Tendermint consensus parameters instead of `x/param`. Legacy types remain in order to facilitate parameter migration from the deprecated `x/params`. App developers should ensure that they execute `baseapp.MigrateParams` during their chain upgrade. These legacy types will be removed in a future release.
 * (client/tx) [#13670](https://github.com/cosmos/cosmos-sdk/pull/13670) Add validation in `BuildUnsignedTx` to prevent simple inclusion of valid mnemonics
+* (mempool) [#13888](https://github.com/cosmos/cosmos-sdk/pull/13888) Mempool implementation that orders tx by nonce and for each select iteration, randomly choose a sender and pick the next nonce ordered tx from their list
```

Comment:
I don't see why we need a changelog for this tbh. The upgrading and varous other docs will point out the existence of a default mempool.

Typically, changelog entries describe changes to existing software (i.e. in a release already) or describe a high-level introduction of a new component. ABCI 1.0 is new, but we dont need a changelog for each piece that relates to it.